src/pages/dmt5/_download-app.tsx

```diff
@@ -30,200 +41,360 @@ const query = graphql`
+    margin-top: 96px;
+
+    @media ${device.tabletL} {
+        max-width: 16px;
+        max-height: 16px;
+        margin-top: 0;
+    }
 `
 const StyledHeader = styled(Header)`
     @media ${device.tabletL} {
         font-size: 20px;
+        text-align: center;
+        margin-top: 30px;
     }
 `
+const StyledFlex = styled(Flex)`
+    justify-content: flex-start;
+    flex-direction: column;
+    max-width: 27rem;
+    margin-right: 100px;
+    height: auto;
+    background-color: var(--color-grey-25);
 
+    @media ${device.laptopL} {
+        max-width: 25rem;
+    }
+
+    @media ${device.tabletL} {
+        justify-content: center;
+        text-align: center;
+        margin-right: 0;
+        max-width: fit-content;
+        box-shadow: inset 0 -2px 0 #e7e7e7;
+    }
+`
+const StyledFlexMobile = styled(Flex)`
+    margin-top: 0.8rem;
+    height: auto;
+    margin-bottom: 40px;
+`
+const StyledFlexBottom = styled(Flex)`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    max-width: 38.4rem;
+
+    @media ${device.tabletL} {
+        padding: 0 160px;
+        background: white;
+        align-items: center;
+        max-width: fit-content;
+    }
+
+    @media ${device.mobileL} {
+        padding: 0 20px;
+        background: white;
+        align-items: center;
+        max-width: fit-content;
+    }
+`
+const StyledInfoContainer = styled(Flex)`
+    display: flex;
+    min-width: 32rem;
+    margin-right: 100px;
+
+    @media ${device.laptopL} {
+        margin-right: 0;
+    }
+
+    @media ${device.laptopM} {
+        margin-right: 0;
+        min-width: 22rem;
+    }
+`
+const StyledHeaderContent = styled(Header)`
+    font-weight: normal;
+    width: 100%;
+    max-width: 30rem;
+    margin-left: 1.6rem;
+    font-size: 24px;
+    margin-top: 96px;
+
+    @media ${device.tabletL} {
+        margin-left: 26px;
+        font-size: 18px;
+        max-width: 33rem;
+        margin-top: 0;
+    }
+`
 const DownloadApp = () => {
     const data = useStaticQuery(query)
 
     return (
         <Section>
-            <Flex
-                jc="flex-start"
-                tablet_jc="center"
-                fd="column"
-                width="100%"
-                max_width="38.2rem"
-                mt="4rem"
-                height="auto"
-                tabletL={{ mt: '0', pl: '10px', pr: '10px' }}
-            >
+            <StyledFlex>
                 <Desktop>
-                    <StyledHeader as="h4" type="subtitle-1">
+                    <StyledHeader as="h4" type="subtitle-1" mt="8px">
                         {localize('Desktop')}
                     </StyledHeader>
                     <Flex mt="0.8rem" jc="flex-start" height="auto">
                         <DownloadLinkWrapper>
                             <LocalizedLink
-                                external
-                                to={dmt5_windows_url}
+                                external="true"
```

Comment:
`external` should be `boolean` please update all of them and check every file for `typescript` issues, thanks thanks :pray: